components/HomePage/LevelButton.vue

```diff
@@ -0,0 +1,16 @@
+<template>
+  <div
```

Comment:
```suggestion
  <button
```
It's better to use the correct tags instead of using `div`s for everything.
The tags help screen readers & other assistive technologies operate. 
I recommend using [MDN's HTML elements reference](https://developer.mozilla.org/en-US/docs/Web/HTML/Element).

Reply:
This still needs to be changed to a button tag